Mathlib/Algebra/Order/Monoid/Unbundled/ExistsOfLE.lean

```diff
@@ -82,4 +82,10 @@ theorem le_iff_forall_one_lt_lt_mul' [MulLeftStrictMono α] :
     a ≤ b ↔ ∀ ε, 1 < ε → a < b * ε :=
   ⟨fun h _ => lt_mul_of_le_of_one_lt h, le_of_forall_one_lt_lt_mul'⟩
 
+set_option linter.docPrime false
```

Comment:
You should add a docstring here instead of disabling the linter.

Reply:
Also, you can try if `le_iff_forall_one_lt_le_mul` is provable by exactly `le_iff_forall_one_lt_le_mul'`. If so, we can remove that lemma and just use `le_iff_forall_one_lt_le_mul` for the name of this lemma.

Reply:
I checked that with the same assumption of [le_iff_forall_one_lt_le_mul](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/Group/DenselyOrdered.html#le_iff_forall_one_lt_le_mul) `exact?` gives `le_iff_forall_one_lt_le_mul'`, so I removed the former and removed `'` from the latter. As `Mathlib.Algebra.Order.Monoid.Unbundled.ExistsOfLE` is imported in `Mathlib.Algebra.Order.Group.DenselyOrdered`, this should cause no problem